ios/TwakeMailNSE/NotificationService.swift

```diff
@@ -155,7 +158,9 @@ class NotificationService: UNNotificationServiceExtension {
         self.modifiedContent?.body = body ?? ""
         self.modifiedContent?.badge = NSNumber(value: badgeCount)
         self.modifiedContent?.sound = .default
-        self.modifiedContent?.userInfo = userInfo
+        var combinedUserInfo = userInfo
+        combinedUserInfo["data"] = originalContent.userInfo
+        self.modifiedContent?.userInfo = combinedUserInfo
```

Comment:
Also please test notification on onBackground

Reply:
FCM background is not affected